<launch>
    <node pkg="erl_geometry" type="gazebo_room_2d_node" name="gazebo_room_2d_node" output="screen">
        <param name="laser_frame" value="front_laser" />
        <param name="map_frame" value="map" />
        <param name="topic_name" value="scan" />
        <param name="publish_rate" value="100.0" />
    </node>
    <node pkg="tf2_ros" type="static_transform_publisher" name="map_odom_transformer"
        args="0 0 0 0 0 0 1 map odom" />
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find erl_geometry)/rviz/gazebo_room_2d.rviz">
    </node>
</launch>
